packages/element-app/src/components/WebKeystore/KeystoreEditor.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line
```

Comment:
the warning says brace should be listed in the project's dependencies. Any reasons why not doing it?

Reply:
Also, looks like brace is not used in this file

Reply:
this is a weired issue with ace... its required.

Reply:
has to do with themes...